### ISSUE TITLE
Training of the network

### ISSUE THREAD

**Issue by Leo63963** [xingyizhou/CenterTrack#231]
Thanks for the great work, really admire that.
I tried to train the network on my own device, with limited GPU (2 * 2080Ti), I have to change the batch_size to 16 to run the training code, and get **UserWarning: was asked to gather along dimension 0, but all input tensors were scalars; will instead unsqueeze and return a vector**. 
It seems this warning is about DataParallel, one says this warning is caused by the wrong input size. 
Besides, the learning rate 1.25e-4 is used for batch size 32, which means I need to change the learning rate accordingly. Just kindly ask, do I need a larger one? 
Thanks.
